src/ThrowsAnnotationReader.php

```diff
@@ -130,9 +130,14 @@ private function getDocblock($reflection): ?string
 	{
 		if ($reflection instanceof MethodReflection) {
 			$declaringClass = $reflection->getDeclaringClass();
-			$nativeClassReflection = $declaringClass->getNativeReflection();
-			$nativeMethodReflection = $nativeClassReflection->getMethod($reflection->getName());
-			$docBlock = $nativeMethodReflection->getDocComment();
+			$classReflection = $declaringClass->getNativeReflection();
+			$methodReflection = $classReflection->getMethod($reflection->getName());
+			$docBlock = $methodReflection->getDocComment();
+
+			while ($docBlock === false) {
+				$methodReflection = $methodReflection->getPrototype();
```

Comment:
It's throwing an exception when a prototype is not there. It should return `null` instead.

Reply:
Yeah, but it's [wrapped in a try-catch block](https://github.com/pepakriz/phpstan-exception-rules/blob/5f30e824dd24cb680a12c365f4244a73bff21b44/src/ThrowsAnnotationReader.php#L97).

Reply:
(except by a try-catch block, there is no way to check whether a method has a prototype)

Reply:
besides the previous comments:
```
 ------ ---------------------------------------------------------------------- 
  Line   src/ThrowsAnnotationReader.php                                        
 ------ ---------------------------------------------------------------------- 
  140    ReflectionException is never thrown in the corresponding try block    
 ------ ---------------------------------------------------------------------- 
```

Reply:
I know, but it's weird in my opinion. `null` value means there is no phpdoc. Thrown exception is for cases when reflection fails.

Reply:
done